viz/charts/linkerd-viz/templates/metrics-api-policy.yaml

```
@@ -0,0 +1,72 @@
+---
+apiVersion: policy.linkerd.io/v1alpha1
+kind: Server
+metadata:
+  namespace: {{.Values.namespace}}
+  name: metrics-api
+  labels:
+    linkerd.io/extension: viz
+    component: metrics-api
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+spec:
+  podSelector:
+    matchLabels:
+      linkerd.io/extension: viz
+      component: metrics-api
+  port: http
+  proxyProtocol: HTTP/1
```

Isn't this a gRPC service?

```suggestion
  proxyProtocol: gRPC
```

---

nm I'm wrong about this. it looks like it's HTTP/1